### PR TITLE
fix #751

### LINKED ISSUE #751: the prep template information shouldn't be collapsed
Currently the prep information in the GUI is collapsed, which is confusing for users. It should always start uncollapsed.

### PR DESCRIPTION
**Pull request by antgonza**
@josenavas @squirrelo @ElDeveloper could you review?

### CODE REVIEW

**Comment by ElDeveloper** [comment]
:+1:

File: qiita_pet/templates/study_description_templates/prep_template_panel.html [hunk 1]
```diff
@@ -44,7 +44,7 @@ <h4 class="panel-title">
       <a data-toggle="collapse" data-parent="#prep-accordion" href="#collapse{{prep_id}}">{{data_type}} (ID: {{prep_id}})</a>
     </h4>
   </div>
-  <div id="collapse{{prep_id}}" class="panel-collapse collapse">
+  <div id="collapse{{prep_id}}" class="panel-collapse collapse in">
```
Comment (squirrelo) [inline qiita_pet/templates/study_description_templates/prep_template_panel.html]: Prettying things up are we? Yay bootstrap animations.

**Comment by squirrelo** [comment]
:+1:

File: qiita_pet/templates/study_description_templates/prep_template_panel.html (hunk 1 above)
Comment (squirrelo) [inline qiita_pet/templates/study_description_templates/prep_template_panel.html]: Oh, this needs the `fade` class before the `in` class to get proper animations.

Comment (antgonza) [inline qiita_pet/templates/study_description_templates/prep_template_panel.html]: I do not feel like is needed but if you feel strong about it, I can add.

Comment (squirrelo) [inline qiita_pet/templates/study_description_templates/prep_template_panel.html]: Ah, only required by alerts. Nevermind,

**Comment by josenavas** [comment]
@squirrelo I think you said that comment can be ignored. If so, can you please merge?

**Pull request merged**